src/content/docs/style-guide/writing-strategies/nav-best-practices.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Docs nav best practices
+metaDescription: "Everything I know about information architecture I learned in library science school."
+---
+
+What is the purpose of a navigation sidebar menu? This is worth considering, because this reader experience is pretty ubiquitous across doc sites, and things that are everywhere tend not to be considered much.
+
+I think a nav menu serves two fundamental purposes:
+
+1. Help readers find the information they're looking for
+2. Give readers insight into the conceptual model of a body of knowledge (in our case, a conceptual model of New Relic)
+
+## Find information [#find]
+
+When I imagine someone finding information, I find that my initial impulse is to picture someone moving in a straight line from question to answer. I feel this impulse, even though I know it to be false!
+
+When human beings move through collections of information—whether it's a docs site, a bookstore, or a temporary collection of information created from an internet search—they do so in a meandering, inefficient, and emotionally impulsive way. Although this process is inefficient, it's the primary way that human beings build conceptual models of understanding!
```

Comment:
This is very helpful. I love the larger insight into the human experience as it applies to this specific example.

Reply:
Glad you like it!